### DISCUSSION TITLE
idea: entering RDF istance will create a voice channel for that istance

### DISCUSSION THREAD

**Discussion by DPComputer** [chromiecraft/chromiecraft#6775] [Suggestions]
i have an idea but i dunno how to realize it:
short elevator pitch: 
a way to integrate discord AND RDF by allowing an easy way to join a voicechannel on CC promptly created by a bot using the istance number as seed
telling in chat "discord channel RDF xxxx has been created" so people can jsut alt-tab here on discord and join

**Comment by heyitsbench** [comment]
Or just implement in-game voice chat. 😛

**Reply by BeefSteaker** [reply]
Thanks for volunteering

**Comment by DPComputer** [comment]
> Or just implement in-game voice chat. 😛

I am no expert at all, but i feel like that would be far more complicated to do, since neither blizz at the time did it.

Also my suggestion aims to be more modular and approachable: something even an addon could manage, by reading the istance number and tell a discord bot what to do with that info.

**Reply by heyitsbench** [reply]
> since neither blizz at the time did it.

Patch 2.2 added in-game voice chat.

**Comment by shaszun** [comment]
Lets waste the time of multiple people to develop a thing, that someone can do it manually in less than a minute. 
https://i.kym-cdn.com/entries/icons/facebook/000/016/729/large.jpg

**Comment by DPComputer** [comment]
> > since neither blizz at the time did it.
> 
> Patch 2.2 added in-game voice chat.

I fairly remember never been used due to not working, in my own experience back in the day, i may be wrong but also never heard the opposite anywhere.

**Comment by DPComputer** [comment]
> Lets waste the time of multiple people to develop a thing, that someone can do it manually in less than a minute.

Was not addressing CC devs in particular, if that's what you refer as "multiple people".
But to whom may concern and reads this pseudo/reddit of a site

And the idea is to make the process even easier and snappier, two clicks away, not even need to alt-tab if you use discord overlay, instead of:

- alt-tab
- open discord and select CC server
- find the voice channel
- maybe other people are there and they already talking or someone else unrelated enters the channel.
- ok that last part was a joke because NOBODY uses CC voice channels, therefore it would promote the use of CC voice channels more!
